build/scoper.inc.php

```diff
@@ -0,0 +1,194 @@
+<?php
+declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+$whitelistClasses = [
+    'PHPUnit\Exception',
+    'PHPUnit\Framework\*',
+    'PHPUnit_Framework_MockObject_MockObject',
+    'SebastianBergmann\CodeCoverage\*',
+    'PharIo\*',
+    'PHP_Token',
+    'PHP_TokenWithScope',
+    'PHP_TokenWithScopeAndVisibility',
+    'PHP_Token_ABSTRACT',
```

Comment:
are those classes or constants?

Reply:
They are classes without any namespace.

Reply:
`php-token-stream` unfortunately does not use namespaces.

@theofidry Would it be possible to add support for whitelisting classes using `PHP_Token_*` or something along those lines?

Reply:
But do they need to be whitelisted?

Reply:
@sebastianbergmann that could be done. I opened https://github.com/humbug/php-scoper/issues/227 for it

Reply:
@theofidry PHP_Token_* classes must be whitelisted, because they are instantiated dynamically (https://github.com/sebastianbergmann/php-token-stream/blob/711ca0c13c66f6b66c2ecb586e56415815034330/src/Token/Stream.php#L182). Prefixing break this.

Reply:
Maybe this could be fixed by having a patcher for [this line](https://github.com/sebastianbergmann/php-token-stream/blob/711ca0c13c66f6b66c2ecb586e56415815034330/src/Token/Stream.php#L176) instead,